Lib/test/test_unicode.py

```diff
@@ -208,7 +208,7 @@ def test_count(self):
 
     def test_find(self):
         string_tests.CommonTest.test_find(self)
-        # test implementation details of the memchr fast path
+        # test implementation details of the std::iter fast path
```

Comment:
Is this in the CPython source?

Reply:
yeah, its okay if we don't change this to say we use `std::iter`.